### ISSUE TITLE
ns_map

### ISSUE THREAD

**Issue by ChristopherTracey** [ropensci/natserv#5]
Hey Scott,
  I noticed the new map functions--very awesome! One bug I noticed is that if there's a county in the distribution that shares its name with counties in other states the data is attributed to those counties as well. So it seems like it needs some sort of county/state combo.
-Chris

**Comment by sckott** [comment]
thanks, will have a look

**Comment by sckott** [comment]
can you give an example?

**Comment by sckott** [comment]
okay, reinstall and try again, I think I fixed the merging problem, and tweaked county map a bit to have grey color polygons for NA'a and low end of the scale is white as before, but now all data should show up better

**Comment by ChristopherTracey** [comment]
Looks better. Unfortunately, I reinstalled the package before saving the map with the errors. I was looking at _Scirpus ancistrochaetus_ (ELEMENT_GLOBAL.2.150401) which only occurs in the northeast US, but it showing presence in counties outside the known range (e.g. Union County, AR was erroneously showing up due to the species being known from Union County, PA). I ran a few species through and the new maps look correct.
